server/neptune/workflows/internal/prrevision/workflow.go

```diff
@@ -17,6 +17,9 @@ import (
 )
 
 const (
+	TaskQueue       = "pr_revision"
```

Comment:
Hmm, why is this defined here, does the workflow need to know about the task queue? Seems like it's the callers responsibility? Would just keep this near the worker

Reply:
> does the workflow need to know about the task queue?

No, but the caller would need to know about the task queue when starting the workflow right since we need to configure the TQ in the `StartWorkflowOptions`. Keeping it in the workflow code makes it easier for the caller to know which TQ to schedule work on, and it also follows the setup we have for the deploy task queue. Lmk if you think otherwise.

Reply:
yeah I think i probably didn't need to keep it in there for the deploy task queue case.  It only gets called from one location so it should have been where the caller was. We can leave this here for consistency.